app/views/account/users/_user.html.slim

```diff
@@ -4,10 +4,10 @@
     - if user.avatar?
       = image_tag(user.avatar.url, alt: 'Image', class: 'user-avatar')
     - else
-      = image_tag("default_a.png", alt: 'Image', class: 'user-avatar')
+      = image_tag('default_a.png', alt: 'Image', class: 'user-avatar')
```

Comment:
can be moved to the UserHelper
```
def user_avatar_tag(user)
  return image_tag('default_a.png', alt: 'Image', class: 'user-avatar') unless user.avatar?
  
  image_tag(user.avatar.url, alt: 'Image', class: 'user-avatar')
end
```
and then in the view
```
 = user_avatar_tag(user)
```

btw, optional

Reply:
done